### ISSUE TITLE
Bus variables in Bus examples not visible since Bus is protected

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#445]
**Reported by otter on 25 Oct 2010 13:07 UTC**
In the examples of:

```
Modelica.Blocks.Examples.BusUsage
Modelica.Mechanics.MultiBody.Examples.Systems.RobotR3
```

an expandable connector is used as a signal bus that is defined as "protected" on highest level. As a result, the bus variables are not stored in the result and the "command script" that plots the results of the bus is no longer valid.

In previous versions this seemed to not appear because the expandable connector was "empty" and the variables have been added if connected to the bus. In MSL3.2, the bus is no longer an empty connector but contains all potential variables on the bus.

(reported by Dag Brück)

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/445

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 25 Oct 2010 13:08 UTC**
Fixed in a001afe5d6f8fd8c996b840633288d2ef1054bb8:

The bus variables of these examples are defined with annotation "HideResult=false".

**Comment by modelica-trac-importer** [comment]
**Modified by otter on 25 Oct 2010 13:15 UTC**
